src/install.js

```diff
@@ -56,11 +57,19 @@ export const DEPENDENCY_FIELDS = [
  * @param  {String} name - name of the dependency.
  * @return {Observable} - observable sequence of `package.json` objects.
  */
-export function resolveLocal (nodeModules, parentTarget, name) {
+export function resolveLocal (nodeModules, parentTarget, name, version) {
 	const linkname = path.join(nodeModules, parentTarget, 'node_modules', name)
 	const fetch = () => EmptyObservable.create()
 	log(`resolving ${linkname} from node_modules`)
 
+	// support `file:` with symlinks
+	if (version.substr(0, 5) === 'file:') {
```

Comment:
I checked, that package doesn't do any http-requests or file operations and does operation synchronously so it can be considered as fast enough to use it instead of writing custom operations. For example, it contains a [fix for Windows](https://github.com/npm/npm-package-arg/blob/master/npa.js#L176-L182) that we have to add as well in future.

It is can be also important when NPM team will add some new features, it will be also available there automatically